### DISCUSSION TITLE
[Question]: Frigate+ Image Tagging

### DISCUSSION THREAD

**Discussion by AntoineGS** [blakeblackshear/frigate#11513] [Ask A Question] [answered]
### What is your question:

I have verified a few hundred images and gotten a custom model with which I am seeing an immense improvement in identification of my Sphynx cats 🤣 (went from about 5% accuracy in tagging them as Cats to 90+%).
I am now running into what I think is multiple objects being identified from the same one (a whole cat vs its head):
![image](https://github.com/blakeblackshear/frigate/assets/16911399/506ef4bf-f256-40d5-b46f-9e592b97e11b)
Seeing that the shorter event is completely part of the the longer one it does not seem to be a min_score issue (might be wrong here but it is based on reading I have done on other discussions).

I noticed that the shorter event is only tagging the head, while the second the whole cat:
![image](https://github.com/blakeblackshear/frigate/assets/16911399/241b5260-a33e-4aaa-bf35-365a0792a67e)
![image](https://github.com/blakeblackshear/frigate/assets/16911399/5ae6135e-ad7a-442d-8f6b-2189d9da99c9)

1) Assuming I am correct, how would I go about correcting the shorter one, do I flag it as 'not a cat' and then add a new tag to a box that has the full cat, or flag is as 'this is a cat' and then resize the box when verifying it?
2) Could this be caused by using images where only the cat head is in the image to train the model?
3) If so, should I only use images where the complete object is in view or tag them as 'difficult' when they are not complete?

Thanks!

**Comment by NickM-27** [comment, marked as answer]
1. Flag as this is a cat and fix the box 
2. Potentially but I wouldn't remove these.
3. Could be considered difficult if only head is visible

**Reply by AntoineGS** [reply]
Thank you!
I am quite amazed at how you and blakeblackshear are active for an open source project, don't burn yourselves out ;)
